docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+services:
+  composer:
+    image: graze/composer
+    volumes:
+      - .:/usr/src/app
+      - ~/.composer:/home/composer/.composer
+
+  php-55:
```

Comment:
When running the coverage tests, you'll see that it complains about xdebug not being installed. I would suggest using the `graze/php-alpine` image which is a smaller image containing some of the extra bits that we use. Also the `test` tag contains xdebug.

To maintain the kind of thing being done in this project, I would recommend this which is derived from the skeleton project.

```yaml
  test:
    image: graze/php-alpine:5-test
    volumes:
      - .:/srv
    working_dir: /srv
```

I can talk you through the changes in more detail and the reasoning behind it if you like.